uptime_service_validation/coordinator/coordinator.py

```diff
@@ -91,12 +91,17 @@ def main():
 
             # Step 5 checks for forks and writes to the db.
             state_hash_df = pd.DataFrame([asdict(submission) for submission in submissions])
-
+            all_files_count = state_hash_df.shape[0]
             if not state_hash_df.empty:
                 master_df["state_hash"] = state_hash_df["state_hash"]
                 master_df["blockchain_height"] = state_hash_df["height"]
                 master_df["slot"] = pd.to_numeric(state_hash_df["slot"])
                 master_df["parent_state_hash"] = state_hash_df["parent"]
+                master_df["submitter"] = state_hash_df["submitter"]
+                master_df["file_updated"] = state_hash_df["submitted_at"]
+                master_df["file_name"] = state_hash_df["submitted_at"]+"-"+state_hash_df["submitter"] #Perhaps this should be changed? Filename makes less sense now.
+                master_df["blockchain_epoch"] = state_hash['created_at'].apply(
```

Comment:
Should this be `state_hash_df['created_at']`?

Reply:
Good spot. Thanks.